packages/app/src/sandbox/compile.ts

```diff
@@ -844,7 +844,7 @@ async function compile(opts: CompileOptions) {
 
     if (manager) {
       const managerState = {
-        ...(await manager.serialize({ optimizeForSize: false })),
+        ...(await manager.serialize()),
```

Comment:
I think we might still want this. The `serialize` here is to send all bundler information to Sandpack components (e.g. if they want to show tests). But for the API cache we definitely want to have a serialized state that's optimized for the API cache storage.

Reply:
Sure, will revert some of these changes